tensorflow/compiler/xla/service/gpu/cudnn_conv_runner.cc

```diff
@@ -334,10 +334,15 @@ StatusOr<CudnnConvParams> GetCudnnConvParams(
   params.window = &conv->window();
   params.dnums = &conv->convolution_dimension_numbers();
   params.feature_group_count = conv->feature_group_count();
+#ifdef GOOGLE_CUDA
```

Comment:
`GOOGLE_CUDA` won't be defined in XLA. can we switch to check only with `TENSORFLOW_USE_ROCM`?